source/includes/_api-reference.md

```diff
@@ -50,18 +58,65 @@ request.AddHeader("Authorization", "Basic [insert credentials here]");
 var request = new RestRequest(Method.GET);
 
 request.AddHeader("Authorization", "ApiKey [insert key here]");
+
+
+// Secure Access Token Authentication
+var request = new RestRequest(Method.GET);
+ 
+request.AddHeader("Authorization", "Bearer [insert Access token here]");
 ```
 
-Intelex utilizes two methods for authentication to the API- Basic and ApiKey.
+Intelex utilizes three methods for authentication to the API- Basic, ApiKey and Secure.
 
 Basic authentication is performed via HTTP Basic Auth and your Intelex user credentials (Username:Password). The credentials are then verified by Intelex and access will be granted or denied accordingly.
 
 ApiKey authentication can be performed by generating API access key(s) from the Intelex User Profile menu (note that this page is only accessible to full-access and admins on the site). 
 The unique generated key can then be used to access the Intelex REST API and used to authenticate and prove that the user is who they claim to be.
 
+Secure token based authentication is supported in V6 API. Only dedicated users can access the API.
+
+During the authentication process, the client provides the ‘client id’ and ‘client secret’ to the authentication endpoint, along with the audience and grant type, as demonstrated above. In response, the API furnishes an access token that comes with a specified time limit. This authentication is exclusively applicable to dedicated users. After successful authentication, users gain access to V6 API endpoints till the token expiration.
+
 All API requests must be made over HTTPS and API requests without authentication will fail. Security on data is managed by the platform security configuration. 
 API requests will maintain the same security settings you have configured in the platform for each user.
 
+
+>auth0 API Endpoint - In the URL the region is the unique region based on the regions they are specifying 
+
+```
+https://intelex-[region]-prd.[region].auth0.com/oauth/token
```

Comment:
where did you get this URL?

Reply:
I do not think it should be like this at all, may instead of the url we can show it on the API access token page for secure api token, i.e. so that users can just copy it from there.

Reply:
I got it from Priyanka

Reply:
![image](https://github.com/IntelexTechnologies/api-docs/assets/128698707/6648c51a-5f35-4d04-8fa4-e2f266448ffe)
Anton, I didn't completely get your suggestion, I see the client id and client secret in API access page in [figma](https://www.figma.com/file/D4lrHisMx2mvkyflB9pU7u/API-Key---V6?type=design&node-id=0-1&mode=design&t=TNnAM68ioDoLQnNm-0 ) 
Can you please elaborate more on this comment.

Reply:
> I do not think it should be like this at all, may instead of the url we can show it on the API access token page for secure api token, i.e. so that users can just copy it from there.

![image](https://github.com/IntelexTechnologies/api-docs/assets/128698707/6648c51a-5f35-4d04-8fa4-e2f266448ffe)
Anton, I didn't completely get your suggestion, I see the client id and client secret in API access page in [figma](https://www.figma.com/file/D4lrHisMx2mvkyflB9pU7u/API-Key---V6?type=design&node-id=0-1&mode=design&t=TNnAM68ioDoLQnNm-0 ) 
Can you please elaborate more on this comment.

Reply:
I'm saying that we probably need to change the UI to show the URL because there is no other way for the user to know it.

Reply:
Do you mean to add **Authorization Endpoint** URL in UI, lets say in mockup between Description and Client ID controls ?